com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -345,8 +345,13 @@ private void Init(bool server)
 
             if (MessageHandler == null)
             {
+                IInternalMessageHandler messageHandler = new InternalMessageHandler(this);
```

Comment:
I'd suggest encapsulating this into a static method to reduce complexity on an already heavy function